engines/grim/grim.cpp

```diff
@@ -841,6 +859,7 @@ void GrimEngine::updateDisplayScene() {
 		updateNormalMode();
 	} else if (_mode == DrawMode) {
 		updateDrawMode();
+		handleUserPaint();
```

Comment:
why this change?

Reply:
I think this might be needed to updating cursor, I'll look closer

Reply:
yes, but you can just press 'I' for inventory

Reply:
Wrong conversation. I don't mind removing it. Remastered doesn't activate the inventory on Ctrl+LMB/MMB.

I think I'll leave middle button though, it can be convenient.

Reply:
ok